trojsten/contests/helpers.py

```diff
@@ -41,3 +43,17 @@ def get_points_from_submits(tasks, submits):
             submit.task, submit.submit_type, submit.user_points, submit.testing_status
         )
     return res
+
+
+def check_description_at_text_submit(user, tasks):
+    user_submits = Submit.objects.latest_for_user(tasks, user)
+    for task in tasks:
+        if (
+            task.has_text_submit
+            and task.has_description
+            and task.round.can_submit
+            and user_submits.filter(task=task, submit_type=SUBMIT_TYPE_TEXT, points__gt=0)
+            and not user_submits.filter(task=task, submit_type=SUBMIT_TYPE_DESCRIPTION)
+        ):
```

Comment:
Toto má potenciál spraviť pomerne veľké množstvo dotazov na databázu.

Reply:
Máš nejaký návrh, ako to spraviť lepšie? Ak to bude iba 1 query a toto finálne filtrovanie sa urobí čisto v Pythone, pomôže to alebo sa to zhorší?

Reply:
@gardenerik Vieš sa, prosím, vyjadriť k tomuto? Ako zefektívniť ten dotaz na databázu?

Reply:
asi by som si vybral všetky submity, ktoré ťa zaujímajú a potom cez nejaký dict zisťovať, či pre danú úlohu (ne)existujú oba